README.md

```diff
@@ -32,6 +32,13 @@ migrator.config({
       collectionName: 'migrations',
       // mongdb url or mongo Db instance
       db: "your connection string",
+      // optional mongdb Client options
```

Comment:
Looks good to me. It would be better to have an expanded db property to hold all the db props/options like below but that would be a breaking change but cleaner. I'm ok having a breaking change now or later. 

```

db: {
  connectionUrl: String;
  name?: String;
  options?:  MongoClientOptions;
}

```

Reply:
Adjusted, see PR update.